osquery/tables/utility/file.cpp

```diff
@@ -33,21 +53,36 @@ const std::map<fs::file_type, std::string> kTypeNames{
     {fs::status_error, "error"},
 };
 
+#endif
+
 void genFileInfo(const fs::path& path,
                  const fs::path& parent,
                  const std::string& pattern,
                  QueryData& results) {
-  // Must provide the path, filename, directory separate from boost path->string
-  // helpers to match any explicit (query-parsed) predicate constraints.
+// Must provide the path, filename, directory separate from boost path->string
+// helpers to match any explicit (query-parsed) predicate constraints.
+
+#if defined(WIN32)
+
+  DWORD dwRtnCode = 0;
```

Comment:
nit: Prefer `unsigned long` over Windows macro types

Reply:
nit: avoid Hungarian notation. We don't have much need for it, specifically in this case, let's take all of these local variables and push their definitions closer to where they're used, right above the function calls, and then change the casing to be `snake_case` for local variables.